tox.ini

```diff
@@ -39,11 +41,20 @@ changedir = {toxinidir}/docs
 commands =
   sphinx-build -W -b html source build/html {posargs}
 
+[testenv:black]
```

Comment:
Can we share the venv here between black and lint by adding a line:
```
envdir = .tox/lint
```

to both job definitions? Right now we're building 2 venvs to just run black while these 2 jobs basically run the same thing (just save a little install time)